optimum/intel/openvino/modeling.py

```diff
@@ -439,6 +443,14 @@ def _from_transformers(
                 raise ValueError("You cannot use both `use_auth_token` and `token` arguments at the same time.")
             token = use_auth_token
 
+        compile_only = kwargs.pop("compile_only", False)
+        if compile_only:
+            logger.warning(
+                "`compile_only` mode will be disabled because it does not support model export."
+                "Please provide openvino model obtained using optimum-cli or saved on disk using `save_pretrained`"
+            )
```

Comment:
please check my comment above, in case if export flag ommited in future, user may not know there is already converted model or not, to not make changes in behaviour after your changes, I stayed warning here